src/Microsoft.Cci.Extensions/Writers/CSharp/CSDeclarationWriter.cs

```diff
@@ -18,6 +18,7 @@ public partial class CSDeclarationWriter : ICciDeclarationWriter
         private bool _forCompilation;
         private bool _forCompilationIncludeGlobalprefix;
         private bool _forCompilationThrowPlatformNotSupported;
```

Comment:
@ericstj do you think it is worth combining these two? With the presence of the message meaning it is enabled?